Update Windows Docker omnibus build instructions (#10948)
DataDog/datadog-agent: docs/dev/agent_omnibus.md

### Prerequisites
To build on Windows, [Docker Desktop](https://docs.docker.com/docker-for-windows/install/) must be installed and configured to use Windows containers.

> Start a Powershell prompt and navigate to your local clone of the `datadog-agent` repo and run:

Reviewer: ```suggestion
Start a Powershell prompt and navigate to your local clone of the `datadog-agent` repo.

 Run the following command: 
 
```

```powershell
docker run -v "$(Get-Location):c:\mnt" -e OMNIBUS_TARGET=main -e RELEASE_VERSION=nightly -e MAJOR_VERSION=7 -e PY_RUNTIMES=3 -e TARGET_ARCH=x64 datadog/agent-buildimages-windows_x64:1809 c:\mnt\tasks\winbuildscripts\buildwin.bat
```

> It takes a while on the first run since it has to download the Docker image.

Reviewer: ```suggestion
Downloading the Docker image may take some time in the first run.
```

> Alternatively here's a small Powershell script to facilitate using the docker image:

Reviewer: ```suggestion
Alternatively, use this Powershell script to help facilitate the Docker image:
```
